src/shared/common/forms/mapped-display.jsx

```diff
@@ -17,6 +17,19 @@ const renderCheckbox = (label, showLabel, choicesLabel) =>  {
     );
 };
 
+const renderCheckboxGroup = (mappings, value, label) => {
+    let displayValue;
```

Comment:
I have simplified down to one line but I don't think replace() can be used since it only replaces the first ',' value it finds when could be multiple that need replacing

Reply:
Actually simplified way further